### PR TITLE
DOCS-2245: Add page about unmasking IBAN

### PR DESCRIPTION
**Pull request by Kris-MultiSafepay**
Do the changes meet MultiSafepay Docs' Definition of Done?
- Style guide, tone of voice, and naming conventions
- Tested
- Reviewed on mobile, tablet and desktop screens
- Notified stakeholders
- Pass GitHub Actions

### CODE REVIEW

File: content/api/orders/post_order_direct.md [hunk 1]
```diff
@@ -164,4 +164,4 @@ __cancel_url__ | string
 Customer will be redirected to this page after a failed payment.
```
Comment (FrancescaNCook) [inline content/api/orders/post_order_direct.md:164]: "If the payment fails, the customer is redirected to this page."

File: content/faq/api/masking-iban-numbers.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
+---
+title : "Unmasking IBAN numbers"
+meta_title: "FAQ API - Unmasking IBAN numbers - MultiSafepay Docs"
+weight: 20
+meta_description: "The MultiSafepay Documentation Center presents all relevant information about our Plugins and API. You can also find support pages for payment methods, tools and general questions as well as the contact details of our Support and Integration Teams."
+read_more: "."
+---
+
+International Bank Account Numbers (IBAN) are sensitive data.
```
Comment (FrancescaNCook) [inline content/faq/api/masking-iban-numbers.md:9]: International bank account numbers

File: content/faq/api/masking-iban-numbers.md [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+title : "Unmasking IBAN numbers"
+meta_title: "FAQ API - Unmasking IBAN numbers - MultiSafepay Docs"
+weight: 20
+meta_description: "The MultiSafepay Documentation Center presents all relevant information about our Plugins and API. You can also find support pages for payment methods, tools and general questions as well as the contact details of our Support and Integration Teams."
+read_more: "."
+---
+
+International Bank Account Numbers (IBAN) are sensitive data. 
+
+By default, we mask IBAN in:
```
Comment (FrancescaNCook) [inline content/faq/api/masking-iban-numbers.md:11]: IBANs

File: content/faq/api/masking-iban-numbers.md [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+
+- [POST notifications](/faq/api/notification-url/#post-notification-example)
+- [GET /orders/{order_id}](/api/#retrieve-an-order) responses
+
+Masked IBAN only display the last 4 digits of the number, e.g. `*** 1234`.
```
Comment (FrancescaNCook) [inline content/faq/api/masking-iban-numbers.md:16]: When masked, only the last 4 digits of the IBAN are visible, e.g.

File: content/faq/api/masking-iban-numbers.md [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+
+### How to unmask
```
Comment (FrancescaNCook) [inline content/faq/api/masking-iban-numbers.md:18]: Unmasking IBANs

File: content/faq/api/masking-iban-numbers.md [hunk 6]
```diff
@@ -0,0 +1,31 @@
...
+
+To unmask IBAN at a website level, reach out to your account manager to discuss why you need to unmask IBAN. You can perform regular business operations like performing refunds without the need of unmasking IBAN.
```
Comment (FrancescaNCook) [inline content/faq/api/masking-iban-numbers.md:20]: You can still perform most business operations with masked IBANs, e.g. processing refunds. 
To unmask IBANs for a specific website, email your reasons to your account manager at <sales@multisafepay.com>

File: content/faq/api/masking-iban-numbers.md [hunk 7]
```diff
@@ -0,0 +1,31 @@
...
+- [GET /orders/{order_id}](/api/#retrieve-an-order) responses
+
+Masked IBAN only display the last 4 digits of the number, e.g. `*** 1234`.
+
+### How to unmask
+
+To unmask IBAN at a website level, reach out to your account manager to discuss why you need to unmask IBAN. You can perform regular business operations like performing refunds without the need of unmasking IBAN. 
+
+
+Once your account manager has approved the unmasking of IBAN, follow these steps:
```
Comment (FrancescaNCook) [inline content/faq/api/masking-iban-numbers.md:23]: Once your account manager has approved your request, follow these steps:

File: content/faq/api/masking-iban-numbers.md [hunk 8]
```diff
@@ -0,0 +1,31 @@
...
+### How to unmask
+
+To unmask IBAN at a website level, reach out to your account manager to discuss why you need to unmask IBAN. You can perform regular business operations like performing refunds without the need of unmasking IBAN. 
+
+
+Once your account manager has approved the unmasking of IBAN, follow these steps: 
+
+1. In your MultiSafepay Control, go to **Settings** > **Website settings**.
+2. Click the relevant website.
+3. Under the **Website functionality** pane, click the **Unmask IBAN numbers in API requests and responses** toggle.
```
Comment (FrancescaNCook) [inline content/faq/api/masking-iban-numbers.md:27]: In the

File: content/faq/api/masking-iban-numbers.md [hunk 9]
```diff
@@ -0,0 +1,31 @@
...
+
+**Note:** The unmask toggle is always visible, however, without the approval of your account manager, it has no effect.
```
Comment (FrancescaNCook) [inline content/faq/api/masking-iban-numbers.md:29]: The unmask toggle is always visible, but only active if approved by your account manager.

File: content/faq/api/masking-iban-numbers.md [hunk 10]
```diff
@@ -0,0 +1,31 @@
...
+
+Unmasked IBAN display the full number, e.g. `NL87ABNA0000001234`.
```
Comment (FrancescaNCook) [inline content/faq/api/masking-iban-numbers.md:31]: When unmasked, the full IBAN is displayed, e.g.

**Review by FrancescaNCook — COMMENTED** [review]
Some suggested tweaks :)

File: content/faq/api/masking-iban-numbers.md (hunk 6 above)
Comment (Kris-MultiSafepay) [inline content/faq/api/masking-iban-numbers.md:20]:
```suggestion
You can still perform most business operations with masked IBANs, e.g. processing refunds.
To unmask IBANs for a specific website, email your reasons to your account manager at <sales@multisafepay.com>
```

File: content/faq/api/masking-iban-numbers.md (hunk 7 above)
Comment (Kris-MultiSafepay) [inline content/faq/api/masking-iban-numbers.md:23]:
```suggestion
Once your account manager has approved your request, follow these steps:
```

File: content/faq/api/masking-iban-numbers.md (hunk 8 above)
Comment (Kris-MultiSafepay) [inline content/faq/api/masking-iban-numbers.md:27]:
```suggestion
3. In the **Website functionality** pane, click the **Unmask IBAN numbers in API requests and responses** toggle.
```

File: content/faq/api/masking-iban-numbers.md (hunk 9 above)
Comment (Kris-MultiSafepay) [inline content/faq/api/masking-iban-numbers.md:29]:
```suggestion
**Note:** The unmask toggle is always visible, but only active if approved by your account manager.
```

File: content/faq/api/masking-iban-numbers.md (hunk 10 above)
Comment (Kris-MultiSafepay) [inline content/faq/api/masking-iban-numbers.md:31]:
```suggestion
When unmasked, the full IBAN is displayed, e.g. `NL87ABNA0000001234`.
```

**Pull request merged**